src/components/landing/LandingFooter.tsx

```diff
@@ -1,3 +1,105 @@
+import styled from "@emotion/styled";
+import Link from "next/link";
+
+import { IcLandingFooter, IcMainLogo } from "../../../public/assets/icons";
+import { DefaultButton } from "../common/styled/Button";
 export default function LandingFooter() {
-  return <div>LandingFooter</div>;
+  return (
+    <>
+      <StFooterWrapper>
+        <StH1>
+          책을 통한 성장의 계단, <br />
+          함께 오르실 분들을 기다립니다.{" "}
+        </StH1>
+        <Link href="/main">
```

Comment:
```suggestion
          <Link href="/main" passHref>
```

맞는지는 모르겠지만 `passHref` 넣어주기?!
[참고 블로그](https://f-dever-error-log.tistory.com/56)

Reply:
수정후에 다시 pr을 날리도록 하겠습니다!